### PR TITLE
[DOC] Update for mkdocs-material 9.5.19 and icons added to Install-LibreNMS.md

### PR DESCRIPTION
**Pull request by santiag0z**
Please give a short description what your pull request is for

image

Reference:
https://squidfunk.github.io/mkdocs-material/reference/icons-emojis/?h=icon

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [ ] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [ ] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.
- [ ] If my Pull Request makes discovery/polling/yaml changes, I have added/updated [test data](https://docs.librenms.org/Developing/os/Test-Units/).

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

**Review by murrant — DISMISSED** [review]

**Comment by murrant** [comment]
You'll need to update the CI build (and documentation) to include the additional modules

**Comment by santiag0z** [comment]
@murrant, I can't resolve the following error:

_Error: MkDocs encountered an error parsing the configuration file: while constructing a Python object
cannot find module 'material.extensions.emoji' (No module named 'material.extensions')
  in "/docs/mkdocs.yml", line 40, column 20
Error: Process completed with exit code 1._

File: .github/workflows/doc.yml [hunk 1]
```diff
@@ -26,6 +26,12 @@ jobs:
       -
         name: Checkout
         uses: actions/checkout@v4
+      - 
+        name: Set up Python runtime
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.x
```
Comment (Jellyfrog) [inline .github/workflows/doc.yml:33]: This is wrong. Extensions are installed inside the dckerrfile.

**Review by Jellyfrog — CHANGES_REQUESTED** [review]

**Comment by santiag0z** [comment]
@murrant and @Jellyfrog 

According to the pymdown-extensions documentation, `pip install pymdown-extensions` was supposed to be sufficient.

Looking at the problem below, sometimes you need `pip install pymdown-extensions` `pip install mkdocs-material-extensions` `pip install mkdocs-material`
https://github.com/facelessuser/pymdown-extensions/issues/110

But I only had success after updating mkdocs-material version 8.3.9 -> 9.5.19

**Review by murrant — APPROVED** [review]
Thanks for updating, looks good now.  I didn't test build this locally, but should work.

**Comment by murrant** [comment]
@santiag0z dang, still didn't build right on CI. https://github.com/librenms/librenms/actions/runs/8884133091/job/24392532294

**Comment by santiag0z** [comment]
putz| I was sad, I'm going to check it calmly and test it first by updating and then with the modification.

**Pull request merged**